yarn-project/end-to-end/src/guides/up_quick_start.sh

```diff
@@ -16,36 +16,37 @@ aztec-cli deploy \
 # docs:end:deploy
 
 aztec-cli check-deploy --contract-address 0x03b030d48607ba8a0562f0f1f82be26c3f091e45e10f74c2d8cebb80d526a69f
+CONTRACT=0x03b030d48607ba8a0562f0f1f82be26c3f091e45e10f74c2d8cebb80d526a69f
```

Comment:
This will not be visible in the docs since it's not included  between the include tags. I think it should probably be there so that user knows where did the var came from.